metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/config/EntityHydratorConfig.java

```diff
@@ -13,19 +13,25 @@
 
 
 @Configuration
-@Import({RestliEntityClientFactory.class, SystemAuthenticationFactory.class})
+@Import({RestliEntityClientFactory.class})
 public class EntityHydratorConfig {
 
   @Autowired
-  @Qualifier("systemAuthentication")
-  private Authentication _systemAuthentication;
+  @Qualifier("systemRestliEntityClient")
+  private SystemRestliEntityClient _entityClient;
 
   @Autowired
-  @Qualifier("restliEntityClient")
-  private RestliEntityClient _entityClient;
+  private EntityRegistry _entityRegistry;
+
+  public final static ImmutableSet<String> EXCLUDED_ASPECTS = ImmutableSet.<String>builder()
```

Comment:
The hydrator is used when writing to datahub usage events. Lineage is known to exceed ES document sizes and is of questionable value for analytics. Excluding various lineage aspects from the DUE entities. The entities themselves are still recorded.